src/PaymentGateways/SettingPage.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace  Give\PaymentGateways;
+
+/**
+ * Interface SettingSection
+ * @package Give\Views\Admin\Settings
+ *
+ * @since 2.8.0
+ */
+interface SettingPage {
+	/**
+	 * Get section id.
+	 * @return string
+	 *
+	 * @since 2.8.0
+	 */
+	public function getId();
+
+	/**
+	 * Get section title.
+	 * @return string
+	 *
+	 * @since 2.8.0
+	 */
+	public function getName();
+
+	/**
+	 * Get section settings.
+	 * @return array
+	 *
+	 * @since 2.8.0
+	 */
+	public function getSettings();
+
+	/**
+	 * Register required properties
+	 *
+	 * @since 2.8.0
+	 */
+	public function register();
```

Comment:
I don't think we need this since we can use the auto injection I mentioned.

Reply:
fixed: https://github.com/impress-org/givewp/compare/8f8e9bd44851...014ff89da09f